vue-frontend/src/components/android-app-development/SuccessStorySection.vue

```diff
@@ -0,0 +1,295 @@
+                :class="[client.className]"
+              >
+                <div
+                  class="tw-font-inter-medium tw-text-[1.125rem] md:tw-text-[2.25rem] lg:tw-text-[3.125rem] tw-leading-[1.6875rem] md:tw-leading-[2.75rem] lg:tw-leading-[3.75rem] tw-transition-all tw-ease tw-duration-500"
+                  v-html="client.review"
+                ></div>
+                <div class="tw-flex tw-flex-row tw-justify-between tw-mt-10">
+                  <div class="tw-flex tw-flex-col">
+                    <span class="tw-flex tw-flex-row">
+                      <font-awesome-icon
+                        v-for="i in 5"
+                        :key="i"
+                        class="fa-star tw-w-[20px] tw-h-[20px] tw-text-[#FF3D2E]"
+                        icon="star"
+                    /></span>
+                    <span
+                      class="tw-mt-3 tw-text-black-core/[0.87] tw-font-inter-semibold tw-text-[1rem] tw-leading-[1.2rem] md:tw-text-[1.625rem]"
+                      >{{ client.name }}</span
+                    >
+                    <span
+                      class="tw-mt-2 tw-font-inter-regular tw-text-[0.75rem] md:tw-text-[1.25rem] tw-leading-[0.9rem] tw-text-black-core/[0.60]"
+                      >{{ client.designation }}</span
+                    >
+                  </div>
+                </div>
+              </div>
+            </swiper-slide>
+          </swiper>
+        </div>
+
+        <div
+          class="tw--mt-4 tw-w-full tw-h-12 after:tw-absolute after:tw-z-20 after:tw-left-0 after:tw-h-[5%] after:tw-w-full after:tw-blur-lg after:tw-bg-white-gradient-bottom"
+        ></div>
+      </div>
+
+      <!-- Mobile UI End-->
+      <div class="tw-hidden md:tw-block lg:tw-mt-20">
+        <h2
+          class="testimonial tw-opacity-[5%] tw-top-1/2 tw-left-1/2 tw-transform tw--translate-x-1/2 tw--translate-y-1/2 tw-absolute tw-tracking-wide tw-font-black tw-font-inter-bold tw-text-[7.5rem] lg:tw-text-[9.5rem] xl:tw-text-[11.75rem] 2xl:tw-text-[13.75rem] xl:tw-leading-[20.625rem] tw-text-black-core/[0.4] tw-text-center"
+        >
+          Testimonials
+        </h2>
+
+        <div
+          class="tw-mx-auto tw-hidden tw-container md:tw-flex tw-flex-row sm:tw-text-black-core/[0.87] tw-text-black-900 md:tw-px-6 lg:tw-px-4 xl:tw-px-24"
+        >
+          <div class="tw-basis-[130%] xl:tw-basis-[85%] tw-pointer-events-none">
+            <swiper
+              :slidesPerView="5"
+              :centeredSlides="true"
+              :centeredSlidesBounds="true"
+              :spaceBetween="10"
+              :slideToClickedSlide="true"
+              :direction="'vertical'"
+              :autoplay="{
+                delay: 4000,
+                disableOnInteraction: false,
+              }"
+              :loop="true"
+              :loopedSlides="50"
```

Comment:
only set scroll on click in aythor profile and auto.
can you able to remove this scrolling ?

https://github.com/canopas/canopas-website/assets/115449373/dff230d5-f02b-4f40-b870-93560a29d20b